### ISSUE TITLE
hooks "_woocommerce_cleanup_draft_orders" with id "0" are created in the table "wp_actionscheduler_actions", one with status "failed" and the other "pending", and the comment "action ignored by Async Request" is added thousands of hundreds of times in the "wp_actionscheduler_logs" table

### ISSUE THREAD

**Issue by camarfbi1** [woothemes/woocommerce#39107]
### Prerequisites

- [X] I have carried out troubleshooting steps and I believe I have found a bug.
- [X] I have searched for similar bugs in both open and closed issues and cannot find a duplicate.

### Describe the bug

Hello,

I don't know what else to do, I've tried several solutions and I can't solve this problem.
I recently had to restore a backup of just my db on my website and since then I've had this problem in woocommerce's scheduled actions.
When creating an order, two hooks "_woocommerce_cleanup_draft_orders" with id "0" are created in the table "wp_actionscheduler_actions", one with status "failed" and the other "pending", and the comment "action ignored by Async Request" is added thousands of hundreds of times in the "wp_actionscheduler_logs" table, my server gets overloaded and my website slows down. I can't stop the scheduled actions, I tried installing the action-scheduler plugin and it broke the site.
can anybody help me?

### Expected behavior

I need woocommerce_cleanup_draft_orders not to be creating endless logs in wp_actionscheduler_logs when creating an order and not finalizing it

### Actual behavior

Hello,

I don't know what else to do, I've tried several solutions and I can't solve this problem.
I recently had to restore a backup of just my db on my website and since then I've had this problem in woocommerce's scheduled actions.
When creating an order, two hooks "_woocommerce_cleanup_draft_orders" with id "0" are created in the table "wp_actionscheduler_actions", one with status "failed" and the other "pending", and the comment "action ignored by Async Request" is added thousands of hundreds of times in the "wp_actionscheduler_logs" table, my server gets overloaded and my website slows down. I can't stop the scheduled actions, I tried installing the action-scheduler plugin and it broke the site.
can anybody help me?
To stop creating logs in wp_actionscheduler_logs, every time I need to run the command in the db: DELETE FROM `wp_actionscheduler_actions` WHERE `status` = 'failed' and 'pending'

### Steps to reproduce

![image](https://github.com/woocommerce/woocommerce/assets/128821400/c9240a59-fb18-4856-81d5-890464119529)

![image](https://github.com/woocommerce/woocommerce/assets/128821400/afde9845-4787-47cd-bf89-0f038395b0a5)

![image](https://github.com/woocommerce/woocommerce/assets/128821400/14432d72-cc34-4ec9-afdf-7999fd9b5951)

### WordPress Environment

`
### WordPress Environment ###

WordPress address (URL): https://canalqualidade.com.br
Site address (URL): https://canalqualidade.com.br
WC Version: 7.8.2
REST API Version: ✔ 7.8.2
WC Blocks Version: ✔ 10.2.4
Action Scheduler Version: ✔ 3.5.4
Log Directory Writable: ✔
WP Version: 6.2.2
WP Multisite: –
WP Memory Limit: 256 MB
WP Debug Mode: –
WP Cron: ✔
Language: pt_BR
External object cache: –

### Server Environment ###

Server Info: Apache/2.4.52 (Debian)
PHP Version: 7.4.27
PHP Post Max Size: 128 MB
PHP Time Limit: 300
PHP Max Input Vars: 1000
cURL Version: 7.74.0
OpenSSL/1.1.1k

SUHOSIN Installed: –
MySQL Version: 5.5.5-10.5.13-MariaDB
Max Upload Size: 128 MB
Default Timezone is UTC: ✔
fsockopen/cURL: ✔
SoapClient: ❌ O seu servidor não possui a classe SoapClient ativada - alguns plugins de sistemas de pagamento que usam SOAP podem não funcionar como esperado.
DOMDocument: ✔
GZip: ✔
Multibyte String: ✔
Remote Post: ✔
Remote Get: ✔

### Database ###

WC Database Version: 7.8.2
WC Database Prefix: wp_
Tamanho total do banco de dados: 94.15MB
Tamanho dos dados do banco de dados: 84.20MB
Tamanho do índice do banco de dado: 9.95MB
wp_woocommerce_sessions: Dados: 0.09MB + Índex: 0.02MB + mecanismo InnoDB
wp_woocommerce_api_keys: Dados: 0.02MB + Índex: 0.03MB + mecanismo InnoDB
wp_woocommerce_attribute_taxonomies: Dados: 0.02MB + Índex: 0.02MB + mecanismo InnoDB
wp_woocommerce_downloadable_product_permissions: Dados: 0.02MB + Índex: 0.06MB + mecanismo InnoDB
wp_woocommerce_order_items: Dados: 0.02MB + Índex: 0.02MB + mecanismo InnoDB
wp_woocommerce_order_itemmeta: Dados: 0.34MB + Índex: 0.11MB + mecanismo InnoDB
wp_woocommerce_tax_rates: Dados: 0.02MB + Índex: 0.06MB + mecanismo InnoDB
wp_woocommerce_tax_rate_locations: Dados: 0.02MB + Índex: 0.03MB + mecanismo InnoDB
wp_woocommerce_shipping_zones: Dados: 0.02MB + Índex: 0.00MB + mecanismo InnoDB
wp_woocommerce_shipping_zone_locations: Dados: 0.02MB + Índex: 0.03MB + mecanismo InnoDB
wp_woocommerce_shipping_zone_methods: Dados: 0.02MB + Índex: 0.00MB + mecanismo InnoDB
wp_woocommerce_payment_tokens: Dados: 0.02MB + Índex: 0.02MB + mecanismo InnoDB
wp_woocommerce_payment_tokenmeta: Dados: 0.02MB + Índex: 0.03MB + mecanismo InnoDB
wp_woocommerce_log: Dados: 0.02MB + Índex: 0.02MB + mecanismo InnoDB
wp_actionscheduler_actions: Dados: 0.02MB + Índex: 0.11MB + mecanismo InnoDB
wp_actionscheduler_claims: Dados: 0.02MB + Índex: 0.02MB + mecanismo InnoDB
wp_actionscheduler_groups: Dados: 0.02MB + Índex: 0.02MB + mecanismo InnoDB
wp_actionscheduler_logs: Dados: 0.36MB + Índex: 0.22MB + mecanismo InnoDB
wp_bp_activity: Dados: 1.52MB + Índex: 0.98MB + mecanismo InnoDB
wp_bp_activity_meta: Dados: 0.19MB + Índex: 0.14MB + mecanismo InnoDB
wp_bp_friends: Dados: 0.02MB + Índex: 0.03MB + mecanismo InnoDB
wp_bp_groups: Dados: 0.02MB + Índex: 0.05MB + mecanismo InnoDB
wp_bp_groups_groupmeta: Dados: 0.02MB + Índex: 0.03MB + mecanismo InnoDB
wp_bp_groups_members: Dados: 0.02MB + Índex: 0.09MB + mecanismo InnoDB
wp_bp_invitations: Dados: 0.02MB + Índex: 0.14MB + mecanismo InnoDB
wp_bp_messages_messages: Dados: 0.02MB + Índex: 0.03MB + mecanismo InnoDB
wp_bp_messages_meta: Dados: 0.02MB + Índex: 0.03MB + mecanismo InnoDB
wp_bp_messages_notices: Dados: 0.02MB + Índex: 0.02MB + mecanismo InnoDB
wp_bp_messages_recipients: Dados: 0.02MB + Índex: 0.08MB + mecanismo InnoDB
wp_bp_notifications: Dados: 0.02MB + Índex: 0.11MB + mecanismo InnoDB
wp_bp_notifications_meta: Dados: 0.02MB + Índex: 0.03MB + mecanismo InnoDB
wp_bp_optouts: Dados: 0.02MB + Índex: 0.05MB + mecanismo InnoDB
wp_bp_user_blogs: Dados: 0.02MB + Índex: 0.03MB + mecanismo InnoDB
wp_bp_user_blogs_blogmeta: Dados: 0.02MB + Índex: 0.03MB + mecanismo InnoDB
wp_bp_xprofile_data: Dados: 0.06MB + Índex: 0.03MB + mecanismo InnoDB
wp_bp_xprofile_fields: Dados: 0.02MB + Índex: 0.08MB + mecanismo InnoDB
wp_bp_xprofile_groups: Dados: 0.02MB + Índex: 0.02MB + mecanismo InnoDB
wp_bp_xprofile_meta: Dados: 0.02MB + Índex: 0.03MB + mecanismo InnoDB
wp_cartflows_ca_cart_abandonment: Dados: 0.02MB + Índex: 0.02MB + mecanismo InnoDB
wp_cartflows_ca_email_history: Dados: 0.02MB + Índex: 0.03MB + mecanismo InnoDB
wp_cartflows_ca_email_templates: Dados: 0.02MB + Índex: 0.00MB + mecanismo InnoDB
wp_cartflows_ca_email_templates_meta: Dados: 0.02MB + Índex: 0.02MB + mecanismo InnoDB
wp_cmplz_cookiebanners: Dados: 0.02MB + Índex: 0.00MB + mecanismo InnoDB
wp_cmplz_cookies: Dados: 0.09MB + Índex: 0.00MB + mecanismo InnoDB
wp_cmplz_services: Dados: 0.02MB + Índex: 0.00MB + mecanismo InnoDB
wp_commentmeta: Dados: 0.02MB + Índex: 0.03MB + mecanismo InnoDB
wp_comments: Dados: 0.09MB + Índex: 0.02MB + mecanismo InnoDB
wp_dlm_downloads: Dados: 0.02MB + Índex: 0.00MB + mecanismo InnoDB
wp_dlm_order: Dados: 0.02MB + Índex: 0.00MB + mecanismo InnoDB
wp_dlm_order_customer: Dados: 0.02MB + Índex: 0.02MB + mecanismo InnoDB
wp_dlm_order_item: Dados: 0.02MB + Índex: 0.02MB + mecanismo InnoDB
wp_dlm_order_transaction: Dados: 0.02MB + Índex: 0.02MB + mecanismo InnoDB
wp_dlm_reports_log: Dados: 0.02MB + Índex: 0.00MB + mecanismo InnoDB
wp_dlm_session: Dados: 0.02MB + Índex: 0.00MB + mecanismo InnoDB
wp_download_log: Dados: 0.02MB + Índex: 0.03MB + mecanismo InnoDB
wp_e_events: Dados: 0.02MB + Índex: 0.02MB + mecanismo InnoDB
wp_e_submissions: Dados: 0.02MB + Índex: 0.27MB + mecanismo InnoDB
wp_e_submissions_actions_log: Dados: 0.02MB + Índex: 0.11MB + mecanismo InnoDB
wp_e_submissions_values: Dados: 0.02MB + Índex: 0.03MB + mecanismo InnoDB
wp_fv_fp_hls_access_tokens: Dados: 0.02MB + Índex: 0.03MB + mecanismo InnoDB
wp_fv_player_encoding_jobs: Dados: 0.02MB + Índex: 0.06MB + mecanismo InnoDB
wp_fv_player_playermeta: Dados: 0.02MB + Índex: 0.03MB + mecanismo InnoDB
wp_fv_player_players: Dados: 0.02MB + Índex: 0.00MB + mecanismo InnoDB
wp_fv_player_videometa: Dados: 0.02MB + Índex: 0.03MB + mecanismo InnoDB
wp_fv_player_videos: Dados: 0.02MB + Índex: 0.03MB + mecanismo InnoDB
wp_h5p_contents: Dados: 0.02MB + Índex: 0.00MB + mecanismo InnoDB
wp_h5p_contents_libraries: Dados: 0.02MB + Índex: 0.00MB + mecanismo InnoDB
wp_h5p_contents_tags: Dados: 0.02MB + Índex: 0.00MB + mecanismo InnoDB
wp_h5p_contents_user_data: Dados: 0.02MB + Índex: 0.00MB + mecanismo InnoDB
wp_h5p_counters: Dados: 0.02MB + Índex: 0.00MB + mecanismo InnoDB
wp_h5p_events: Dados: 0.02MB + Índex: 0.00MB + mecanismo InnoDB
wp_h5p_libraries: Dados: 0.08MB + Índex: 0.03MB + mecanismo InnoDB
wp_h5p_libraries_cachedassets: Dados: 0.02MB + Índex: 0.00MB + mecanismo InnoDB
wp_h5p_libraries_hub_cache: Dados: 0.02MB + Índex: 0.02MB + mecanismo InnoDB
wp_h5p_libraries_languages: Dados: 3.52MB + Índex: 0.00MB + mecanismo InnoDB
wp_h5p_libraries_libraries: Dados: 0.02MB + Índex: 0.00MB + mecanismo InnoDB
wp_h5p_results: Dados: 0.02MB + Índex: 0.02MB + mecanismo InnoDB
wp_h5p_tags: Dados: 0.02MB + Índex: 0.00MB + mecanismo InnoDB
wp_h5p_tmpfiles: Dados: 0.02MB + Índex: 0.02MB + mecanismo InnoDB
wp_links: Dados: 0.02MB + Índex: 0.02MB + mecanismo InnoDB
wp_myCRED_log: Dados: 0.02MB + Índex: 0.09MB + mecanismo InnoDB
wp_options: Dados: 6.47MB + Índex: 0.00MB + mecanismo InnoDB
wp_pmpro_discount_codes: Dados: 0.02MB + Índex: 0.05MB + mecanismo InnoDB
wp_pmpro_discount_codes_levels: Dados: 0.02MB + Índex: 0.02MB + mecanismo InnoDB
wp_pmpro_discount_codes_uses: Dados: 0.02MB + Índex: 0.03MB + mecanismo InnoDB
wp_pmpro_memberships_categories: Dados: 0.02MB + Índex: 0.02MB + mecanismo InnoDB
wp_pmpro_memberships_pages: Dados: 0.02MB + Índex: 0.02MB + mecanismo InnoDB
wp_pmpro_memberships_users: Dados: 0.06MB + Índex: 0.09MB + mecanismo InnoDB
wp_pmpro_membership_levelmeta: Dados: 0.02MB + Índex: 0.03MB + mecanismo InnoDB
wp_pmpro_membership_levels: Dados: 0.02MB + Índex: 0.05MB + mecanismo InnoDB
wp_pmpro_membership_ordermeta: Dados: 0.02MB + Índex: 0.03MB + mecanismo InnoDB
wp_pmpro_membership_orders: Dados: 0.02MB + Índex: 0.20MB + mecanismo InnoDB
wp_postmeta: Dados: 31.52MB + Índex: 2.58MB + mecanismo InnoDB
wp_posts: Dados: 12.23MB + Índex: 0.28MB + mecanismo InnoDB
wp_revslider_css: Dados: 0.13MB + Índex: 0.02MB + mecanismo InnoDB
wp_revslider_css_bkp: Dados: 0.02MB + Índex: 0.02MB + mecanismo InnoDB
wp_revslider_layer_animations: Dados: 0.02MB + Índex: 0.00MB + mecanismo InnoDB
wp_revslider_layer_animations_bkp: Dados: 0.02MB + Índex: 0.00MB + mecanismo InnoDB
wp_revslider_navigations: Dados: 0.02MB + Índex: 0.00MB + mecanismo InnoDB
wp_revslider_navigations_bkp: Dados: 0.02MB + Índex: 0.00MB + mecanismo InnoDB
wp_revslider_sliders: Dados: 0.02MB + Índex: 0.02MB + mecanismo InnoDB
wp_revslider_sliders_bkp: Dados: 0.02MB + Índex: 0.02MB + mecanismo InnoDB
wp_revslider_slides: Dados: 0.02MB + Índex: 0.02MB + mecanismo InnoDB
wp_revslider_slides_bkp: Dados: 0.02MB + Índex: 0.02MB + mecanismo InnoDB
wp_revslider_static_slides: Dados: 0.02MB + Índex: 0.02MB + mecanismo InnoDB
wp_revslider_static_slides_bkp: Dados: 0.02MB + Índex: 0.02MB + mecanismo InnoDB
wp_signups: Dados: 0.05MB + Índex: 0.06MB + mecanismo InnoDB
wp_termmeta: Dados: 0.02MB + Índex: 0.03MB + mecanismo InnoDB
wp_terms: Dados: 0.05MB + Índex: 0.03MB + mecanismo InnoDB
wp_term_relationships: Dados: 0.06MB + Índex: 0.05MB + mecanismo InnoDB
wp_term_taxonomy: Dados: 0.05MB + Índex: 0.03MB + mecanismo InnoDB
wp_tm_taskmeta: Dados: 0.02MB + Índex: 0.03MB + mecanismo InnoDB
wp_tm_tasks: Dados: 0.02MB + Índex: 0.02MB + mecanismo InnoDB
wp_usermeta: Dados: 2.52MB + Índex: 0.72MB + mecanismo InnoDB
wp_users: Dados: 0.08MB + Índex: 0.05MB + mecanismo InnoDB
wp_wc_admin_notes: Dados: 0.06MB + Índex: 0.00MB + mecanismo InnoDB
wp_wc_admin_note_actions: Dados: 0.05MB + Índex: 0.02MB + mecanismo InnoDB
wp_wc_category_lookup: Dados: 0.02MB + Índex: 0.00MB + mecanismo InnoDB
wp_wc_customer_lookup: Dados: 0.02MB + Índex: 0.03MB + mecanismo InnoDB
wp_wc_download_log: Dados: 0.02MB + Índex: 0.03MB + mecanismo InnoDB
wp_wc_order_coupon_lookup: Dados: 0.02MB + Índex: 0.03MB + mecanismo InnoDB
wp_wc_order_product_lookup: Dados: 0.02MB + Índex: 0.06MB + mecanismo InnoDB
wp_wc_order_stats: Dados: 0.02MB + Índex: 0.05MB + mecanismo InnoDB
wp_wc_order_tax_lookup: Dados: 0.02MB + Índex: 0.03MB + mecanismo InnoDB
wp_wc_product_attributes_lookup: Dados: 0.02MB + Índex: 0.02MB + mecanismo InnoDB
wp_wc_product_download_directories: Dados: 0.02MB + Índex: 0.02MB + mecanismo InnoDB
wp_wc_product_meta_lookup: Dados: 0.02MB + Índex: 0.09MB + mecanismo InnoDB
wp_wc_rate_limits: Dados: 0.02MB + Índex: 0.02MB + mecanismo InnoDB
wp_wc_reserved_stock: Dados: 0.02MB + Índex: 0.00MB + mecanismo InnoDB
wp_wc_tax_rate_classes: Dados: 0.02MB + Índex: 0.02MB + mecanismo InnoDB
wp_wc_webhooks: Dados: 0.02MB + Índex: 0.02MB + mecanismo InnoDB
wp_wfblockediplog: Dados: 0.28MB + Índex: 0.00MB + mecanismo InnoDB
wp_wfblocks7: Dados: 0.02MB + Índex: 0.05MB + mecanismo InnoDB
wp_wfconfig: Dados: 2.52MB + Índex: 0.00MB + mecanismo InnoDB
wp_wfcrawlers: Dados: 0.02MB + Índex: 0.00MB + mecanismo InnoDB
wp_wffilechanges: Dados: 0.02MB + Índex: 0.00MB + mecanismo InnoDB
wp_wffilemods: Dados: 11.55MB + Índex: 0.00MB + mecanismo InnoDB
wp_wfhits: Dados: 1.28MB + Índex: 0.22MB + mecanismo InnoDB
wp_wfhoover: Dados: 0.02MB + Índex: 0.02MB + mecanismo InnoDB
wp_wfissues: Dados: 0.02MB + Índex: 0.06MB + mecanismo InnoDB
wp_wfknownfilelist: Dados: 5.52MB + Índex: 0.00MB + mecanismo InnoDB
wp_wflivetraffichuman: Dados: 0.02MB + Índex: 0.02MB + mecanismo InnoDB
wp_wflocs: Dados: 0.02MB + Índex: 0.00MB + mecanismo InnoDB
wp_wflogins: Dados: 0.41MB + Índex: 0.13MB + mecanismo InnoDB
wp_wfls_2fa_secrets: Dados: 0.02MB + Índex: 0.02MB + mecanismo InnoDB
wp_wfls_role_counts: Dados: 0.00MB + Índex: 0.00MB + mecanismo MEMORY
wp_wfls_settings: Dados: 0.02MB + Índex: 0.00MB + mecanismo InnoDB
wp_wfnotifications: Dados: 0.02MB + Índex: 0.00MB + mecanismo InnoDB
wp_wfpendingissues: Dados: 0.02MB + Índex: 0.06MB + mecanismo InnoDB
wp_wfreversecache: Dados: 0.02MB + Índex: 0.00MB + mecanismo InnoDB
wp_wfsnipcache: Dados: 0.02MB + Índex: 0.05MB + mecanismo InnoDB
wp_wfstatus: Dados: 0.17MB + Índex: 0.11MB + mecanismo InnoDB
wp_wftrafficrates: Dados: 0.02MB + Índex: 0.00MB + mecanismo InnoDB
wp_wfwaffailures: Dados: 0.02MB + Índex: 0.00MB + mecanismo InnoDB
wp_wpforms_tasks_meta: Dados: 0.02MB + Índex: 0.00MB + mecanismo InnoDB
wp_wpmailsmtp_debug_events: Dados: 0.02MB + Índex: 0.00MB + mecanismo InnoDB
wp_wpmailsmtp_tasks_meta: Dados: 0.02MB + Índex: 0.00MB + mecanismo InnoDB
wp_yoast_indexable: Dados: 0.13MB + Índex: 0.13MB + mecanismo InnoDB
wp_yoast_indexable_hierarchy: Dados: 0.02MB + Índex: 0.05MB + mecanismo InnoDB
wp_yoast_migrations: Dados: 0.02MB + Índex: 0.02MB + mecanismo InnoDB
wp_yoast_primary_term: Dados: 0.02MB + Índex: 0.03MB + mecanismo InnoDB
wp_yoast_seo_links: Dados: 0.09MB + Índex: 0.09MB + mecanismo InnoDB

### Post Type Counts ###

attachment: 352
bp-email: 98
certificate: 6
condition_group: 1
course: 10
course-card: 4
course-layout: 6
dlm_download: 5
dlm_download_version: 5
e-landing-page: 1
elementor_library: 5
flamingo_contact: 376
flamingo_inbound: 68
forum: 2
group-card: 2
group-layout: 2
guest-author: 2
member-card: 2
member-profile: 3
mycred_rank: 1
nav_menu_item: 109
news: 1
oembed_cache: 2
page: 48
popups: 1
post: 15
product: 8
question: 30
quiz: 11
restriction: 1
shop_coupon: 17
shop_order: 105
shop_order_refund: 1
topic: 1
tour: 28
unit: 96
user_request: 1
vibe_zoom: 9
vibecal_event: 5
whatsapp-accounts: 2
wp_global_styles: 1
wpcf7_contact_form: 3
wpforms: 1
wplms-assignment: 2

### Security ###

Secure connection (HTTPS): ✔
Hide errors from visitors: ✔

### Active Plugins (47) ###

Query Monitor: por John Blackbourn – 3.12.3
Advanced Cron Manager: por BracketSpace – 2.5.0
Advanced Database Cleaner: por Younes JFR. – 3.1.2
bbPress: por Contribuidores do bbPress – 2.6.9
BuddyPress: por A comunidade BuddyPress – 11.2.0
Editor clássico: por Colaboradores do WordPress – 1.6.3
Co-Authors Plus: por Mohammad Jangda
Daniel Bachhuber
Automattic – 3.5.10

Contact Form 7: por Takayuki Miyoshi – 5.7.7
Download Monitor: por WPChill – 4.8.6
Duplicate Page: por mndpsingh287 – 4.5.2
Elementor: por Elementor.com – 3.14.1
Flamingo: por Takayuki Miyoshi – 2.4
FV Player Pro: por Foliovision – 7.5.35.7212
FV Player: por  – 7.5.36.7212
Site Kit by Google: por Google – 1.104.0
Loco Translate: por Tim Whitlock – 2.6.4
Meta pixel for WordPress: por Facebook – 3.0.12
Paid Memberships Pro: por Paid Memberships Pro – 2.10.5
PMPro Customizations: por Paid Memberships Pro – .1
Post Category Image With Grid and Slider: por WP OnlineSupport
Essential Plugin – 1.4.8

Really Simple SSL: por Really Simple Plugins – 7.0.5
reCAPTCHA for WooCommerce: por Elliot Sowersby
RelyWP – 1.3.0

Advanced Editor Tools: por Automattic – 5.9.0
Vibe Calendário: por VibeThemes – 1.4
Meu Drive: por VibeThemes – 1.4.2
Ganhos: por VibeThemes – 1.8.1
Vibe HelpDesk: por VibeThemes – 1.3.4
Vibe Zoom: por VibeThemes – 1.8.8
VibeBP: por VibeThemes – 1.9.8.4.1
Editor de campo de checkout para WooCommerce: por ThemeHigh – 1.9.0
WooCommerce Parcelas Com e Sem Juros: por Leonardo Robles – 1.6.1
Mercado Pago payments for WooCommerce: por Mercado Pago – 6.9.2
Claudio Sanches - PagSeguro for WooCommerce: por Claudio Sanches – 2.14.0
WooCommerce: por Automattic – 7.8.2
Wordfence Security: por Wordfence – 7.10.0
Yoast SEO: por Equipe Yoast – 20.10
WP Add Custom CSS: por Daniele De Santis – 1.1.6
Super Page Cache for Cloudflare: por Optimole – 4.7.4
WP Mail SMTP: por WP Mail SMTP – 3.8.0
WP Chat App: por NinjaTeam – 3.4.2
WPLMS Batches: por Vibethemes – 4.9.2
Complemento WPLMS CoAuthor Plus: por VibeThemes – 4.0
WPLMS Customizer Plugin: por VibeThemes – 1.0
WPLMS PDF Certificates: por VibeThemes – 2.7
WPLMS S3: por Mr.Vibe – 4.3
WPLMS: por VibeThemes – 1.9.7.3

### Inactive Plugins (4) ###

Action Scheduler: por Automattic – 3.6.1
Action Scheduler: por Automattic – 3.6.1
Cleanup Action Scheduler: por Media Uganda – 1.1.0
Redis Object Cache: por Till Krüss – 2.4.3

### Dropin Plugins (2) ###

advanced-cache.php: advanced-cache.php
db.php: Query Monitor Database Class (Drop-in)

### Settings ###

API Enabled: –
Force SSL: –
Currency: BRL (R$)
Currency Position: left
Thousand Separator: .
Decimal Separator: ,
Number of Decimals: 2
Taxonomies: Product Types: external (external)
grouped (grouped)
simple (simple)
variable (variable)

Taxonomies: Product Visibility: exclude-from-catalog (exclude-from-catalog)
exclude-from-search (exclude-from-search)
featured (featured)
outofstock (outofstock)
rated-1 (rated-1)
rated-2 (rated-2)
rated-3 (rated-3)
rated-4 (rated-4)
rated-5 (rated-5)

Connected to WooCommerce.com: ✔
Enforce Approved Product Download Directories: –
HPOS feature screen enabled: –
HPOS feature enabled: –
Order datastore: WC_Order_Data_Store_CPT
HPOS data sync enabled: –

### WC Pages ###

Base da loja: #29 - /shop/
Carrinho: #30 - /cart/
Finalização de compra: #31 - /checkout/
Minha conta: #32 - /my-account/
Termos e condições: #33 - /politica-de-reembolsos-e-devolucoes/

### Theme ###

Name: WPLMS Blank ChildTheme
Version: 1.0
Author URL: http://themeforest.com/user/vibethemes
Child Theme: ✔
Parent Theme Name: WPLMS
Parent Theme Version: 4.900
Parent Theme Author URL: http://www.VibeThemes.com/
WooCommerce Support: ✔

### Templates ###

Overrides: wplms-child/woocommerce/archive-product.php
wplms/woocommerce/cart/cart.php versão 7.4.0 está desatualizado. A versão do core é 7.8.0
wplms/woocommerce/checkout/form-checkout.php
wplms/woocommerce/content-product.php
wplms/woocommerce/content-single-product.php
wplms/woocommerce/single-product/short-description.php
wplms/woocommerce/single-product/title.php
wplms/woocommerce/single-product.php

Outdated Templates: ❌
					
					
						Aprenda como atualizar

### Admin ###

Enabled Features: activity-panels
analytics
product-block-editor
coupons
customer-effort-score-tracks
import-products-task
experimental-fashion-sample-products
shipping-smart-defaults
shipping-setting-tour
homescreen
marketing
mobile-app-banner
navigation
onboarding
onboarding-tasks
remote-inbox-notifications
remote-free-extensions
payment-gateway-suggestions
shipping-label-banner
subscriptions
store-alerts
transient-notices
woo-mobile-welcome
wc-pay-promotion
wc-pay-welcome-page

Disabled Features: core-profiler
minified-js
new-product-management-experience
product-variation-management
settings
async-product-editor-category-field

Daily Cron: ✔ Next scheduled: 2023-07-06 13:00:18 -03:00
Options: ✔
Notes: 104
Onboarding: completed

### Action Scheduler ###

Concluído: 32
Oldest: 2023-06-04 21:45:03 -0300
Newest: 2023-07-04 18:44:28 -0300

Malsucedido: 1
Oldest: 2023-07-05 17:09:01 -0300
Newest: 2023-07-05 17:09:01 -0300

Pendente: 1
Oldest: 2023-07-05 17:09:01 -0300
Newest: 2023-07-05 17:09:01 -0300

### Status report information ###

Generated at: 2023-07-05 17:51:16 -03:00
`

### Isolating the problem

- [X] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [X] This bug happens with a default WordPress theme active, or [Storefront](https://woocommerce.com/storefront/).
- [X] I can reproduce this bug consistently using the steps above.

**Comment by SirRahikkala** [comment]
I will chime in and ask if this is related to issue where stock management and hold order timer is not working. Our stores are not changing order status from pending payment to cancelled after set time.

**Comment by camarfbi1** [comment]
I ended up restoring the db again and using the previous version of woocomercce and it solved the problem, I believe that some data had been corrupted in the previous import.

**Comment by AndrewJDawes** [comment]
Hey, @camarfbi1 !

I have encountered this exact issue before - and yes, it was caused by a "corrupted" database table that failed to restore properly (after the restore, a table was missing an `auto_increment PRIMARY KEY` constraint). It was **not** caused by any bug in WooCommerce.

For example, here is the SQL I used to restore primary keys on the `sbm_options` table (note the custom prefix and change as needed).

https://gist.github.com/AndrewJDawes/be798b463f20d9a80c021133dd8b22ed

Again, I needed to do this when the `auto_increment PRIMARY KEY` constraints disappeared after an attempted database restore. The result of the missing constraints was the table got "corrupted" (WordPress was inserting duplicate primary keys in the options table - many with value `0`. This was breaking stuff.

I would recommend closing this issue since it was a problem with the environment and not WooCommerce.
